src/GovernorCountingFractional.sol

```diff
@@ -53,6 +56,14 @@ abstract contract GovernorCountingFractional is Governor {
         return _proposalVotersWeightCast[proposalId][account] > 0;
     }
 
+    /**
+     * @dev Get the number of votes cast on proposal `proposalId` by account `account`.
```

Comment:
Nit: add "so far" to this, i.e. "the number of votes cast **so far** on proposal"